integration-tests/switch-wallet.test.ts

```diff
@@ -2,20 +2,57 @@ import {
   cleanup,
   createAccounts,
   createNewWallet,
+  getDocumentsByType,
+  setNetwork,
   setupEnvironent,
 } from './helpers';
+import {BasicCredential} from './data/credentials';
+import {importCredentialJSON} from './helpers/credential-helpers';
+
+async function createWalletData() {
+  await createAccounts();
+  await importCredentialJSON(BasicCredential);
+}
+
+async function assertWalletData() {
+  const accounts = await getDocumentsByType('Address');
+  const credentials = await getDocumentsByType('VerifiableCredential');
+
+  expect(accounts.length).toBe(2);
+  expect(credentials.length).toBe(1);
+}
+
+async function isWalletEmpty() {
+  const accounts = await getDocumentsByType('Address');
+  const credentials = await getDocumentsByType('VerifiableCredential');
+
+  return !(accounts.length && credentials.length);
+}
 
 describe('Switch wallet', () => {
-  beforeAll(async () => {
+  beforeEach(async () => {
     await cleanup();
     await setupEnvironent();
     await createNewWallet();
-    await createAccounts();
   });
 
-  it('expect to import data in the mainnet wallet', () => {});
+  it('expect to filter mainnet documents from testnet documents', async () => {
+    await setNetwork('mainnet');
+    await createWalletData();
+    await assertWalletData();
+    await setNetwork('testnet');
+    // const isEmpty = await isWalletEmpty();
```

Comment:
what's happening with this commented code?

Reply:
fixed, should be running now